less/index.less

```diff
@@ -1,7 +1,47 @@
+// === Variables === //
+
+// Background Colors
+@background: @TanHide;
+@ctabg: @CreamCan;
+@headerbg: @CreamCan;
+@contentbg: @CatalinaBlue;
+@contactbg: @CreamCan;
+@footerbg: @TanHide;
+
+// Tablet BG Colors
+// Font Colors
+@fontcolor: black;
+
+// Color Bank
+@Olive: #c7d59f;
+@SoftGreen: #b7ce63;
+@CalmGreen: #8fb339;
+@DarkGreen: #4b5842;
+@TanHide: #f78764;
+@CreamCan: #f4d35e;
+@Cabaret: #da4167;
+@WhiteRock: #ebebd3;
+@CatalinaBlue: #083d77;
+
+
+// === Mixins === //
+
+// Enter desired pixel size in parameter
+// mixin will convert to rems
+.fontsizeConverter(@pixsize) {
```

Comment:
lol, I don't think we would actually use this instead of just changing it to `rems`, but this is super, super cool. It probably would have been a great idea back in the `em` days.